### PR TITLE
add .mailmap to fix authors for git-shortlog

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#
+# This list is used by git-shortlog to fix a few botched name translations
+# in the git archive, either because the author's full name was messed up
+# and/or not always written the same way, making contributions from the
+# same person appearing not to be so.
+#
+
+Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
```
Comment (pedrobaeza) [inline .mailmap:8]: Why is this two times?

**Comment by pedrobaeza** [comment]
Is this so needed? It's not enough having GitHub account linked to that emails?

File: .mailmap (hunk 1 above)
Comment (yvaucher) [inline .mailmap:8]: The second line is there to assign @gurneyalex with another mail to his main mail.
Doing git shortlog will then regroup his contributions. git shortlog doesn't care about github accounts.

**Comment by pedrobaeza** [comment]
OK then :+1:

**Comment by omar7r** [comment]
:+1:

**Comment by guewen** [comment]
:+1:

**Pull request merged**